### ISSUE TITLE
Can't create custom keyboard shortcuts

### ISSUE THREAD

**Issue by YamiYukiSenpai** [pop-os/cosmic-settings#669]
I tried to add a keyboard shortcut for Yakuake using Ctrl+Alt+Y, and it won't let me via custom shortcuts

the input field seems to just take in whatever I'm typing, and not the key combination.  In this scenario, I wanted to use `Ctrl+Alt+Y` as the shortcut, but it doesn't register those keys, but I can type "Ctrl+Alt+Y" for some reason, and set it.  However, doesn't mean the key registered, but I found it funny.

**Comment by MikMik1011** [comment]
+1

**Comment by Toniolo-Marco** [comment]
I am experiencing the same bug.

**Comment by YamiYukiSenpai** [comment]
Hmm..now it worked
I just did Ctrl+Alt+Y

**Comment by jannisbecker** [comment]
Thank you for the hint about case sensitivity.
I wanted to use Super+Space to open the launcher, like many other operating systems do,
and I couldn't get it to accept it. Typing Super+space worked perfectly now..

**Comment by mmstick** [comment]
Yeah we will eventually modify this to accept raw key codes once possible
